src/adapters/index.ts

```diff
@@ -316,3 +329,16 @@ export const supportedProtocols: Record<
     [Chain.Linea]: [XfaiDexAdapter],
   },
 }
+
+export const WriteActionInputs = {
+  AaveV3ATokenWriteActionInputs,
+  CompoundV2BorrowMarketWriteActionInputs,
+  CompoundV2SupplyMarketWriteActionInputs,
+}
+
+export const GetTransactionParamsSchema = z.union([
+  AaveV3ATokenGetTransactionParamsSchema,
+  CompoundV2BorrowMarketGetTransactionParamsSchema,
+  CompoundV2SupplyMarketGetTransactionParamsSchema,
+])
+export type GetTransactionParams = z.infer<typeof GetTransactionParamsSchema>
```

Comment:
just to align: this is updated automatically when a GetTransactionParams type is exported from an adapter file?

Reply:
We need to run `npm run build-types`